spec/controllers/applications_controller_spec.rb

```diff
@@ -24,7 +23,7 @@
     context 'with a non lease up application' do
       let(:expected_non_lease_up_app) { fixture('controllers/applications/non_lease_up_application_domain.json') }
 
-      it 'should return a domain application' do
+      it 'should return a domain application snapshot' do
```

Comment:
Hm, I think with a non-lease up application is should *not* return a snapshot, right?

Reply:
Good catch, updated.